kubernetes/host-logs-geneva.yaml

```diff
@@ -56,10 +56,13 @@ spec:
        livenessProbe:
           exec:
             command:
-              - cmd
-              - /c
-              - .\opt\hostlogswindows\scripts\cmd\livenessprobe.exe
-              - ".\\etc\\hostlogswindows\\filesystemwatcher.txt"
+              - C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
+              - -command
+              - |
+                .\opt\hostlogswindows\scripts\powershell\livenessprobe.ps1 -exeRelativePath ".\\opt\\genevamonitoringagent\\genevamonitoringagent\\Monitoring\Agent\\MonAgentManager.exe" -fileName ".\\etc\\hostlogswindows\\filesystemwatcher.txt"
```

Comment:
the exeRelativePath isn't working for me unless I remove ".\\\\" at the start

Reply:
Fixed